exporter/metrics_exporter.go

```diff
@@ -29,16 +30,31 @@ type MetricsExporter interface {
 }
 
 // MultiMetricsExporters wraps multiple metrics exporters in a single one.
-func MultiMetricsExporters(mes ...MetricsExporter) MetricsExporter {
+func MultiMetricsExporters(mes ...MetricsExporter) receiver.MetricsReceiverSink {
 	return metricsExporters(mes)
 }
 
 type metricsExporters []MetricsExporter
 
+var _ receiver.MetricsReceiverSink = (*metricsExporters)(nil)
+
 // ExportMetricsData exports the MetricsData to all exporters wrapped by the current one.
 func (mes metricsExporters) ExportMetricsData(ctx context.Context, md data.MetricsData) error {
 	for _, me := range mes {
 		_ = me.ExportMetricsData(ctx, md)
 	}
 	return nil
 }
+
+// ReceiveTraceData receives the span data in the protobuf format, translates it, and forwards the transformed
```

Comment:
nit: our friend golint is not gone like that the comment doesn't match with the function name :)

Reply:
Haha, gotcha.